src/app/login/loginForm.tsx

```diff
@@ -56,30 +58,30 @@ export const LoginForm: NextPage<Props> = ({ providers, session }) => {
         </Button>
       </form>
       <div className="flex items-center justify-center pt-4">
-        
-        {providers !== null && Object.values(providers)?.map(
-          (item: any) =>
-            item.id !== "credentials" && (
-              <div className="justify-content ml-3 flex" key={item.id}>
-                <Button
-                  color="primary"
-                  variant="outlined"
-                  aria-label="outlined primary button group"
-                  onClick={async () =>
-                    await signIn(item.id, { callbackUrl: "/dashboard" })
-                  }
-                >
-                  <Image
-                    alt={item.id}
-                    src={`https://authjs.dev/img/providers/${item.id}.svg`}
-                    height={50}
-                    width={50}
-                  />
-                  <span className="pl-2">{item.id}</span>
-                </Button>
-              </div>
-            ),
-        )}
+        {providers !== null &&
+          Object.values(providers)?.map(
+            (item: ItemProvider) =>
```

Comment:
You can skip type here, so it will infer it by itself. Please try it like this:
```suggestion
            (item) =>
```